app/src/main/java/in/testpress/testpress/ui/ProductDetailsActivity.java

```diff
@@ -0,0 +1,152 @@
+package in.testpress.testpress.ui;
+
+import android.graphics.PorterDuff;
+import android.os.Bundle;
+import android.support.v4.app.LoaderManager;
+import android.support.v4.content.Loader;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.ListAdapter;
+import android.widget.ListView;
+import android.widget.ProgressBar;;
+import android.widget.RelativeLayout;
+import android.widget.TextView;
+
+import com.nostra13.universalimageloader.core.DisplayImageOptions;
+import com.nostra13.universalimageloader.core.ImageLoader;
+
+import java.net.URL;
+import java.util.Arrays;
+
+import javax.inject.Inject;
+
+import butterknife.ButterKnife;
+import butterknife.InjectView;
+import in.testpress.testpress.Injector;
+import in.testpress.testpress.R;
+import in.testpress.testpress.TestpressServiceProvider;
+import in.testpress.testpress.models.ProductDetails;
+import in.testpress.testpress.util.FormatDate;
+import retrofit.RetrofitError;
+
+public class ProductDetailsActivity extends TestpressFragmentActivity implements LoaderManager.LoaderCallbacks<ProductDetails>{
+
+    @Inject TestpressServiceProvider serviceProvider;
+    @InjectView(R.id.thumbnail_image) ImageView image;
+    @InjectView(R.id.title) TextView titleText;
+    @InjectView(R.id.number_of_exams) TextView numberOfExamsText;
+    @InjectView(R.id.number_of_notes) TextView numberOfNotesText;
+    @InjectView(R.id.date) TextView dateText;
+    @InjectView(R.id.categories) TextView categoriesText;
+    @InjectView(R.id.price) TextView priceText;
+    @InjectView(R.id.description_title) TextView descriptionTitleText;
+    @InjectView(R.id.description) TextView descriptionText;
+    @InjectView(R.id.examsTitle) TextView examsTitleText;
+    @InjectView(R.id.examListView) ListView examListViewText;
+    @InjectView(R.id.notesTitle) TextView notesTitleText;
+    @InjectView(R.id.notesListView) ListView notesListViewText;
+    @InjectView(R.id.productDetails) RelativeLayout productDetailsView;
```

Comment:
There is no need to add Text suffix in the variable names. Try to name it simple hereafter unless it is conflicting. For `examsListView` means it is a list of exams while `exams` means it is a list of `exam` models.